### DISCUSSION TITLE
Newbie questions: Benchmarking /Mining on Unmineable, Ethermine, F2pool, Hiveon and Ravenminer

### DISCUSSION THREAD

**Discussion by HERusr** [RainbowMiner/RainbowMiner#1698] [General]
Hi All,
I'm a newbie to mining and would like to know how to mine on unmineable for coins, I have enable it on the pool's but when I run a benchmark both on CPU and GPU, it doesn't seems to query that site. the same happens on Ethermine, F2pool, Hiveon and Ravenminer, the pools are loaded when the program starts but on benchmark it doesn't show up query/testing these pools. pls. help.

,

,
,

**Comment by RainbowMiner** [comment]
Could you please upload a Debug file? Just open http://localhost:4000 in your browser and click "Debug file" on the left hand side. I can then check your configuration.

**Reply by HERusr** [reply]
[debug_2021-08-12.zip](https://github.com/RainbowMiner/RainbowMiner/files/6977959/debug_2021-08-12.zip)
Attached as requested.

**Reply by RainbowMiner** [reply]
Your setup looks perfectly allright - one thing, you might want to check: open `http://localhost:4000` and check "All Pools" page, if the pools are showing up. If yes, you have the situation, that other pools are more profitable.

This is how RainbowMiner is doing it's job:
- each round, it first selects the most profitable pool for each algorithm. That selection can be found on page "Best Pools".
- It then assigns the best (fastest) miner to the best pools selection.
- third, it starts the miners on the selected pools.

If you want to get your rig mining on a specific pool, try to reduce the number of selected pools ("PoolName" list in config.txt). In my experience, a selection of 3-5 pools is perfect. I am currently mining on Ethermine (for some ETH and ETC), Nicehash, ProHashing (for some LTC), unMineable (for some MATIC and XTZ), ZergPoolCoins and rent out my rig on MiningRigRentals.
